tests/unit/test_class_cmds.py

```diff
@@ -1541,72 +1532,3 @@ class command that can be tested.  This includes:
         """
         self.command_test(desc, self.command_group, inputs, exp_response,
                           mock, condition, verbose=False)
-
```

Comment:
Why have these tests been removed?

Reply:
Sorry, forgot to add comment.  Turns out that they were the original tests I wrote when starting the project and added nothing to the overall testing except to make it harder to isolate to a single test.  All that functionality is incorporated into the set of test cases.   I had meant to drop them long ago and forgot.  Should have made it a separate PR.

Reply:
ok with that.